contracts/src/v0.8/interfaces/MigratableKeeperRegistryInterfaceV2.sol

```diff
@@ -0,0 +1,26 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.0;
+
+import "../UpkeepFormat.sol";
+
+interface MigratableKeeperRegistryInterfaceV2 {
+  /**
+   * @notice Migrates upkeeps from one registry to another, including LINK and upkeep params.
+   * Only callable by the upkeep admin. All upkeeps must have the same admin. Can only migrate active upkeeps.
+   * @param upkeepIDs ids of upkeeps to migrate
+   * @param destination the address of the registry to migrate to
+   */
+  function migrateUpkeeps(uint256[] calldata upkeepIDs, address destination) external;
+
+  /**
+   * @notice Called by other registries when migrating upkeeps. Only callable by other registries.
+   * @param encodedUpkeeps abi encoding of upkeeps to import - decoded by the transcoder
+   */
+  function receiveUpkeeps(bytes calldata encodedUpkeeps) external;
+
+  /**
+   * @notice Specifies the version of upkeep data that this registry requires in order to import
+   */
+  function upkeepVersion() external returns (uint8 version);
```

Comment:
i'm wondering if we should create new interfaces just for this change. Another way we can solve the issue is by adding V1, V2, ... V256 in the enum values which has the same effect and it doesn't require new interfaces. wdyt?

Reply:
actually on reviewing further I see you changed the name too which makes the distinction clear in v2.0 registry